nidaqmx/_grpc_stub_interpreter.py

```diff
@@ -0,0 +1,394 @@
+import threading
+import warnings
+
+import grpc
+import nidaqmx.nidaqmx_pb2 as grpc_types
+import nidaqmx.nidaqmx_pb2_grpc as nidaqmx_grpc
+import nidaqmx.session_pb2 as session_grpc_types
+
+class GrpcStubInterpreter(object):
+    '''Interpreter for interacting with a gRPC Stub class'''
+
+    def __init__(self, grpc_options):
+        self._grpc_options = grpc_options
+        self._lock = threading.RLock()
```

Comment:
nimi-python uses this to implement the IVI lock/unlock functions. DAQmx does not have an equivalent user-accessible lock, so delete this.